group/init/v2/schema.js

```diff
@@ -19,16 +19,27 @@ module.exports = {
     groupKey: { $ref: '#/definitions/groupKey' },
     tangles: {
       type: 'object',
-      required: ['group', 'members'],
+      required: ['group', 'members', 'epoch'],
+      additionalProperties: false,
       properties: {
-        group: { $ref: '#/definitions/tangle/root' },
+        group: { $ref: '#/definitions/tangle/any' },
+        epoch: { $ref: '#/definitions/tangle/any' },
         members: { $ref: '#/definitions/tangle/root' }
       }
+    },
+    recps: {
+      type: 'array',
+      items: [
+        { $ref: '#/definitions/groupId' },
+        { $ref: '#/definitions/feedId' }
+      ],
+      minItems: 2,
+      maxItems: 2
```

Comment:
:fire:
this `recps` is only possible for `group/init-epoch`. So it's good that it's not a `required`, but I think it's gonna be confusing for people reading this unless we split these. It's subtle that you don't know the `groupId` till after the groupRoot is created.